flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -184,7 +184,9 @@ def read_parameters_from_config(modinf: model_info.ModelInfo):
 
                 else:
                     raise ValueError(
-                        f"unsure how to read outcome {new_comp}: not a str, nor an incidence or prevalence: {src_name}"
+                        f"Unable to read outcome '{new_comp}': "
+                        f"expected a string or dictionary containing `incidence` or `prevalence`. "
+                        f"Recieved: '{src_name}'."
```

Comment:
```suggestion
                        f"Expected a string or dictionary containing 'incidence' or 'prevalence', instead given '{src_name}', for outcome '{new_comp}'."
```
Trying to shorten this and clean up the grammar, open to other suggestions on how to best do that.